integration_test/ops_agent_test/main_test.go

```diff
@@ -4732,12 +4739,13 @@ func TestNoNvmlOtelReceiverWithoutGpu(t *testing.T) {
 			t.Fatal(err)
 		}
 
-		time.Sleep(60 * time.Second)
-		_, err := gce.QueryLog(ctx, logger, vm, "syslog", time.Hour, `jsonPayload.message=~"Nvidia|nvml"`, 5)
-		if err == nil {
-			t.Error("expected no logs to contain Nvidia or nvml when the instance has no gpu")
-		} else if !strings.Contains(err.Error(), "not found, exhausted retries") {
-			t.Fatalf("unexpected error: %v", err)
+		otelConfig, err := retrieveOtelConfig(ctx, logger, vm)
+		if err != nil {
+			t.Fatal(err)
+		}
+
+		if strings.Contains(otelConfig, "nvml") {
+			t.Errorf("expects the nvml receiver not in the Otel configuration for VMs without GPU. Otel config: " + otelConfig)
```

Comment:
[optional][nit] Similar to the previous comment, here `t.Errorf` is not necessary if we are concatenating the strings. I suggest to use only one, formatting with `Errorf` or string concatenation.

Reply:
Done!